src/Qir/Runtime/build-qir-runtime.ps1

```diff
@@ -105,7 +105,7 @@ $osQirDropFolder = Join-Path $Env:DROPS_DIR QIR $os
 if (!(Test-Path $osQirDropFolder)) {
     New-Item -Path $osQirDropFolder -ItemType "directory"
 }
-$pattern | Foreach-Object { Copy-Item (Join-Path . bin $_) $osQirDropFolder }
+$pattern | Foreach-Object { Copy-Item (Join-Path . bin $_) $osQirDropFolder -ErrorAction SilentlyContinue}
```

Comment:
Why do we want to silently continue on errors?

Reply:
It's a subtle but important reason. If the above generate step fails, then the source `bin` folder might not exist, in which case this line will throw. If it throws, then we won't execute the `Pop-Location` below, and the script execution will leave the present working directory as the folder in the subtree. This doesn't matter for the pipeline, as a failure will kill the powershell session and propagate up to a failure of the whole pipeline, but for local development it's extremely annoying to have a script fail and leave you in a random directory. There are two ways to handle this: wrap the copy in a try-catch or suppress the throw. Given that the copy already doesn't look for specific files and the failure only happens when the `bin` folder is missing completely, we know a line higher up in the script has already failed with the more relevant failure, so no need to distract with another throw here. Another alternative could be to put the whole script execution in a try-catch and have every failure throw, ensuring that the `Pop-Location` is in the finally block, but I erred on the side of keeping the overall structure of the script the same.

This goes along with my effort to make sure the build scripts work for local dev as well as they do for the pipeline, so I wanted to make sure to fix this to smooth out the experience.